pandas/io/data.py

```diff
@@ -400,26 +405,26 @@ def get_data_yahoo(symbols=None, start=None, end=None, retry_count=3,
     ----------
     symbols : string, array-like object (list, tuple, Series), or DataFrame
         Single stock symbol (ticker), array-like object of symbols or
-        DataFrame with index containing stock symbols.
+        DataFrame with index containing stock symbols, default: None
```

Comment:
Yes.  On line 406 instead (not sure but it looks like that's how other function parameters are documented)?